components/channel_view/channel_view.tsx

```diff
@@ -81,24 +89,25 @@ export default class ChannelView extends React.PureComponent {
 
         return null;
     }
-
-    constructor(props) {
+    public channelViewRef: any;
```

Comment:
I'm seeing all the methods public and I think is not necessary right? also avoid using any and try to put the type here